File: Frontend/DisasterDispatchFrontend/src/app/Models/DTOS/EmergencyReport/emergency-report-dto.ts
import { Guid } from "guid-typescript";
import { CustomOperation } from "../../custom-operation";

export class EmergencyReportDto {
    id: Guid;
    reporterTc: string;
    reporterPhoneNumber: string;
    info: string;
    province: string;
    district: string;
    neighbourhood: string;
    street: string;
    status: string;
    customOperations: CustomOperation[]
}

File: Backend/DisasterDispatch.Core/Dtos/EmergencyReportDtos/EmergencyReportDto.cs
using DisasterDispatch.Core.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DisasterDispatch.Core.Dtos.EmergencyReportDtos
{
    public class EmergencyReportDto
    {
        public Guid Id { get; set; }
        public string ReporterTc { get; set; }
        public string ReporterPhoneNumber { get; set; }
        public string Info { get; set; }
        public string Province { get; set; }
        public string District { get; set; }
        public string Neighbourhood { get; set; }
        public string Street { get; set; }
        public string Status { get; set; }
        public ICollection<CustomOperation> CustomOperations { get; set; }        
    }
}
